### DISCUSSION TITLE
CLIP for ReID model

### DISCUSSION THREAD

**Discussion by yinmayoo185** [mikel-brostrom/yolo_tracking#1025] [General]
Will there be add CLIP as reid model in this boxmot?

**Comment by mikel-brostrom** [comment]
This (https://github.com/Syliz517/CLIP-ReID) you mean?

**Reply by yinmayoo185** [reply]
I am actually trying to add this one https://github.com/DeepSportradar/player-reidentification-challenge

**Reply by yinmayoo185** [reply]
I mean this one https://github.com/DeepSportradar/2022-winners-player-reidentification-challenge

**Reply by mikel-brostrom** [reply]
CLIP-ReID implemented (https://arxiv.org/abs/2211.13977) and working here: https://github.com/mikel-brostrom/yolo_tracking/pull/1031

**Comment by mikel-brostrom** [comment]
It would make sense to add heavier ReID model to the repo...
